tests/test_compression_wrapper.c

```diff
@@ -27,7 +27,7 @@
 #include "createrepo/misc.h"
 #include "createrepo/compression_wrapper.h"
 
-#define TMP_FILE_PATTERN                        "test_XXXXXX.txt"
```

Comment:
This broke when I added this line https://github.com/rpm-software-management/createrepo_c/pull/391/files#diff-c87d8ed02804b9ad227639fcb6469f0bba1256da4cf73f1937a6d023bfd374b9R179